.travis.yml

```diff
@@ -106,61 +61,16 @@ jobs:
               - docker exec -it travis_coverity_scan coverity.sh build
               - docker exec -it travis_coverity_scan coverity.sh upload
 
-        - stage: clean docker
-          language: python
-          # python:
-              # - "3.6"  Probably broken ATM
-          env:
-              - SIZE_LIMIT="3000" # Limit in MBs
-              - TAG_LIMIT="3" # Number of tags to be kept at the time
-          before-script:
-              - sudo apt-get -y install python3
-          script:
-                # Get docker-remote tool and setup venv
-              - sudo $CI_TOOL_DIR/get-docker-remote.sh
```

Comment:
`get-docker-remote.sh` doesn't seem to be used any more, so it probably should be deleted.